Documentation/deployment-and-configuration.md

```diff
@@ -46,6 +46,40 @@ etcd_certfile=/etc/ssl/etcd/client.pem
 etcd_keyfile=/etc/ssl/etcd/client-key.pem
 ```
 
+### Basic Authentication
+
+If your etcd cluster has [Basic authentication][etcd-authentication] enabled, you will need to configure fleet to use an username/password combination for a valid user in the system. Also, because [Basic authentication][etcd-authentication] is Base64 encoded and easily deciphered, it is recommended to also use [TLS authentication][etcd-security] for transport level encryption by providing an `etcd_cafile`. *Authentication is only available since etcd 2.1.X and greater.*
+The examples below show how to achieve this:
+
+#### Using systemd Drop-Ins
+
+```ini
+[Service]
+Environment="FLEET_ETCD_SERVERS=https://192.0.2.12:2379"
+Environment="FLEET_ETCD_USERNAME=root"
+Environment="FLEET_ETCD_PASSWORD=coreos"
+```
+
+#### Using CoreOS Cloud Config
```

Comment:
@tixxdz doc already contains info on how to use tls auth: https://github.com/coreos/fleet/blob/master/Documentation/deployment-and-configuration.md#tls-authentication